### ISSUE TITLE
Bitrate and freqdev should be in kbps and kHz

### ISSUE THREAD

**Issue by leobel96** [Velocet/iown-homecontrol#23]
From
https://github.com/Velocet/iown-homecontrol/blob/95778c3be582d035dee93f8811e9e7bd53b7bb3c/src/IoHome.cpp#L26
and
https://github.com/Velocet/iown-homecontrol/blob/95778c3be582d035dee93f8811e9e7bd53b7bb3c/src/IoHome.cpp#L27
the values should be 38.4 and 19.2

**Comment by Velocet** [comment]
Why use a higher unit which needs floats instead of this simple define?

**Comment by leobel96** [comment]
Define? I'm just saying that, from the Radiolib documentation:
```
struct FSKRate_t {
  /*! \brief FSK bit rate in kbps */
  float bitRate;
  
  /*! \brief FS frequency deviation in kHz*/
  float freqDev;
};
```

So, considering that you want 38.4kbps and 19.2kHz the values should be 38.4 and 19.2

**Comment by Velocet** [comment]
You are totally right! Shame on me! Will be fixed with the next bigger code commit.

Thank you for finding and correcting me 👍🏻

**Comment by leobel96** [comment]
No problem :)
